test/e2e/autorepair.go

```diff
@@ -0,0 +1,145 @@
+package e2e
```

Comment:
If we want to be able to have a unit test job that does `go test ./...` we need to keep the build tags

Reply:
Agree, this is why I added the tag in the first place

Reply:
fixed